data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -145,6 +145,18 @@ datasets:
         field_type: bool
         include_in_tiles: false
         include_in_downloadable_files: false
+  - long_name: "Abandoned Mine Land Inventory System"
+    short_name: "eAMLIS"
+    module_name: "eamlis"
+    load_fields:
+      - short_name: "has_amlis"
+        df_field_name: "BINARY_HAS_AMLIS"
+        long_name: "Is there at least one abandoned mine in this census tract?"
+        description_short: 
+          "Whether the tract has an abandoned mine"
+        field_type: bool
+        include_in_tiles: false
```

Comment:
I think these should both be true? Because it comes into the score. As far as I can tell, this doesn't currently matter, though

Reply:
Cool, will change, and thank you.

Reply:
Thank you! I always feel like I'm guessing at these --- done in 702804e39047697fd3673a64fdd56256dfd82d58